src/firefly/js/visualize/saga/CoverageWatcher.js

```diff
@@ -111,7 +111,7 @@ export function* watchCoverage({viewerId, options= {}}) {
 
         switch (action.type) {
 
-            case TABLE_NEW:
+            case TABLE_LOADED:
                 if (!getTableInGroup(tbl_id)) continue;
                 decimatedTables[tbl_id]= null;
                 displayedTableId = updateCoverage(tbl_id, viewerId, decimatedTables, options);
```

Comment:
Should we update coverage on invoked_by=TABLE_SORT? (Since all it is doing is collecting points to calculate coverage image.)